### PR TITLE
Use updated link in setup.cfg

### PR DESCRIPTION
**Pull request by ZhengnanZhao**
- [ ] Tests added / passed
- [ ] Passes `black dask` / `flake8 dask`

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,9 +1,7 @@
 [flake8]
 # References:
 # https://flake8.readthedocs.io/en/latest/user/configuration.html
-# https://flake8.readthedocs.io/en/latest/user/error-codes.html
-
-# Note: there cannot be spaces after comma's here
```
Comment (ZhengnanZhao) [inline setup.cfg:6]: This note is no longer accurate. Spaces are now allowed after commas.

File: setup.cfg [hunk 2]
```diff
@@ -2,8 +2,7 @@
 # References:
 # https://flake8.readthedocs.io/en/latest/user/configuration.html
 # https://flake8.readthedocs.io/en/latest/user/error-codes.html
-
-# Note: there cannot be spaces after comma's here
+# https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
```
Comment (ZhengnanZhao) [inline setup.cfg:5]: The existing link, `https://flake8.readthedocs.io/en/latest/user/error-codes.html`, only covers error codes starting with `F`. This additional link contains error codes starting with `E` and `W` as well.

The existing link directs readers to the new link, as seen below.
> Users should also reference pycodestyle’s list of error codes.

File: setup.cfg (hunk 1 above)
Comment (ZhengnanZhao) [reply setup.cfg:6]: Spaces _are_ currently being used after commas.
https://github.com/dask/dask/blob/a4c571475b1c643dedf99bc773c2564bf3fcf977/setup.cfg#L6-L11

**Comment by mrocklin** [comment]
Thanks @ZhengnanZhao !

**Pull request merged**